internal/rest/resources/resources.go

```diff
@@ -95,6 +95,15 @@ func ValidateEndpoints(extensionServers map[string]rest.Server, coreAddress stri
 			serverAddresses[server.Address.String()] = true
 		}
 
+		// Reserve the /core namespace for Microcluster
+		if server.CoreAPI {
```

Comment:
Worth adding another line to the doc string of `ValidateEndpoints` to document the new behavior?